EIPS/eip-3076.md

```diff
@@ -0,0 +1,292 @@
+---
+eip: 3076
+title: Validator client interchange format (slashing protection)
+author: Michael Sproul (@michaelsproul), Sacha Saint-Leger (@sachayves), Danny Ryan (@djrtwo)
+discussions-to: https://discord.com/channels/595666850260713488/598292067260825641/
```

Comment:
This is the first time someone has proposed a Discord channel as a discussion-to link, so I'll need to talk with the other editors to see if that is allowed.  Normally, an https://ethereum-magicians.org/ forum post is the preferred discussion-to location.

Reply:
So I realise this was a strange choice, but I couldn't think of a better one. 

From EIP-1:

>  examples for places to discuss your EIP include Ethereum topics on Gitter, an issue in this repo or in a fork of this repo, Ethereum Magicians (this is suitable for EIPs that may be contentious or have a strong governance aspect), and Reddit r/ethereum.

Since this EIP is neither contentious nor has a strong governance aspect, I didn't think it was worth starting an  Ethereum Magicians thread. 

Since (I assume?) Discord channels perform much the same role as Gitter topics did back then, and the discussion has already naturally occurred there (see https://discord.com/channels/595666850260713488/598292067260825641/745665794435973170), I thought it might be appropriate.

Although, I have to say, I'm not completely happy with it, since the url is less expressive than a Gitter topic url, and one has to join the entire discord group to join the channel.

Reply:
We've started an Ethereum Magicians thread nonetheless, as it's nice to have a focussed discussion, whereas the previous `eth2` channel covers a lot of other topics and is likely to be noisy:

https://ethereum-magicians.org/t/eip-3076-validator-client-interchange-format-slashing-protection/

Reply:
Replaced discord with ethereum magicians link. We can mark this as resolved?